원재/Week2_ReverseString.js

```diff
@@ -0,0 +1,7 @@
+function solution(my_string) {
+    // Array.from 을 이용하여 유사 배열 객체를 배열로 바꿈
+    const array = Array.from(my_string);
```

Comment:
Array.from까지~ 좋네요

해당 방법도 있겠지만, 간단하게 수정해보자면
```suggestion
    const array = [...my_string];
```
도 똑같은 의미입니다! spread 연산자까지 추가로 학습하시면 더 좋을거같네요